poetry/masonry/builders/builder.py

```diff
@@ -86,6 +86,8 @@ def find_excluded_files(self):  # type: () -> Set[str]
 
         explicitely_excluded = set()
         for excluded_glob in self._package.exclude:
+            if os.path.isdir(os.path.join(self._path.as_posix(), str(excluded_glob))):
```

Comment:
Can you extract the argument from `os.path.isdir` here and put it in a variable inside this `for`-loop since we are also using the same argument for `glob` in line 92?